frontend/src/lib/components/instances/ComparisonView.svelte

```diff
@@ -180,7 +180,7 @@
 </script>
 
 <div class="h-full w-full overflow-auto" bind:this={instanceContainer}>
-	<table class="mt-2 w-full table-fixed">
```

Comment:
Why the min-w? Shouldn't this just scale and scroll anyway?

Reply:
The instance views shrink to be kind of tiny in some cases. This forces them to at least fill most of the space.

Reply:
Should we rather set a min-w on the instance view? This will not really solve the problem, right? If the table really is less than 1000 it's kinda weird and if there are tons of columns the instance will still shrink to tiny.

Reply:
Hmm don't think I understand. Some instance views can be tiny, do we want to make them bigger than they have to be?